dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf

```diff
@@ -14,6 +14,23 @@ resource "kafka_topic" "pubsub_examples" {
   }
 }
 
+resource "kafka_topic" "pubsub_examples_2" {
```

Comment:
Whats this?

Reply:
Sorry it was just here to verify change. and didn't remove to cause confusion .. 
now removed.